database/seed_data.js

```diff
@@ -16,15 +15,61 @@ var dummyDataGenerator = () => {
     
         return arrayOfReservations.sort();
     };
-
     let reservations = dateGenerator();
-    console.log(reservations);
-
-    // all users/listings will have the same dates reserved / save time
     for (let i = 101; i < 201; i++) {
-        var user = {id: i, dates_reserved: reservations};
+        let user = {id: i, minStay: Math.floor(Math.random() * 3) + 1, dates_reserved: reservations};
```

Comment:
maybe you were generating users since we needed 100 records, but i think that you just need available dates, so i don't think the dates need to be attached to any other data.